### PR TITLE
[MIG] barcodes_generator_location: Migration to 12.0

### PR DESCRIPTION
**Pull request by chienandalu**
This module expands Odoo functionality, allowing user to generate barcode
depending on a given barcode rule for Stock Locations.

cc @Tecnativa TT21485

### CODE REVIEW

**Comment by chienandalu** [comment]
@pedrobaeza @sergio-teruel Can you review?

File: barcodes_generator_location/models/barcode_rule.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class BarcodeRule(models.Model):
+    _inherit = 'barcode.rule'
+
+    generate_model = fields.Selection(
+        selection_add=[('stock.location', 'Stock Location')],
+    )
```
Comment (sergio-teruel) [inline barcodes_generator_location/models/barcode_rule.py:11]: Maybe you need create an uninstall hook for set this field to other or blank...

**Review by sergio-teruel — APPROVED** [review]
Only a comment

**Review by victoralmau — APPROVED** [review]

File: barcodes_generator_location/models/barcode_rule.py (hunk 1 above)
Comment (pedrobaeza) [reply barcodes_generator_location/models/barcode_rule.py:11]: @chienandalu please take into account this remark.

Comment (chienandalu) [reply barcodes_generator_location/models/barcode_rule.py:11]: I opted for delete them directly

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**